docs/experimental-features.md

```diff
@@ -705,3 +706,42 @@ ipfs config --json Experimental.StrategicProviding true
     - [ ] provide roots
     - [ ] provide all
     - [ ] provide strategic
+
+## Filesystem API Plugin
+
+### State
+
+Experimental, not included by default.
+
+This daemon plugin wraps the IPFS node and exposes file system services over a multiaddr listener.  
+Currently we use the 9P2000.L protocol, and offer read-only support for `/ipfs` requests. With support for other (writable) systems coming next.  
+
+You may connect to this service using the `v9fs` client used in the Linux kernel.
+By default we listen locally on the machine, using a Unix domain socket.  
+`mount -t 9p -o trans=unix $IPFS_PATH/filesystem.9p.sock ~/ipfs-mount`  
```

Comment:
s/9p/9P/

Reply:
I'm with you on this, but the implementation wants it this way. (at least on my machine)
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/plain/Documentation/filesystems/9p.txt
![image](https://user-images.githubusercontent.com/13862850/65077637-5efdde00-d969-11e9-8af5-6b4916fa7844.png)

Also hello 👋!!

Reply:
Sorry I meant the `filesystem.9p.sock` should be `filesystem.9P.sock`. Hello! :wave:

Reply:
Ahh, nice catch. I'm going to go in the opposite direction, anywhere I have a 'P' is going to be lowered to 'p' to be consistent with the mount argument.